book/content/open-source-comms/intro.md

```diff
@@ -0,0 +1,16 @@
+# Community Communications for Open Source Proects
+
+## Summary
+
+> Conversation as Collaboration
+
+In an Open Source project, a significant amount of collaboration between the core or leadership team and the wider community takes place in the form of online interactions (likely many of them!).
+It is therefore really important to plan carefully _how_ and _where_ these interactions take place. 
+Community interactions should maintain transparency and openness of the project to facilitate the growth of your community, and the choice of the platforms should ensure that your project is welcoming, accessible, and secure for its collaborators.
+
+This chapter gives an overview of the most common ways to track, support and maintain community interactions such as by raising issues, reporting bugs and attracting contributions that lead to the improvements in the project.
+
+## How this will help you/ why this is useful
+
+When trying to build a community around an Open Source project, the content in this chapter will help make sure that your communications are as open, transparent and accessible as possible.
+This will help foster trust between the project team and the wider community and give them the agency to contribute where they can.
```

Comment:
Might be worth adding a sentence explicitly saying what the risks associated with not doing this well are? Not sure!